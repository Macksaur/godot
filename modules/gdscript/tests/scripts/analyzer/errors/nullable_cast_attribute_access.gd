class A:
    var foo: String = ""
func test():
    var nullable_instance: A? = null
    (nullable_instance as A).foo # Should not error
    (nullable_instance as A?).foo # Should Error