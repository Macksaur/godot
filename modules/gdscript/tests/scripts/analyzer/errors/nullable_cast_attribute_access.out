GDTEST_ANALYZER_ERROR
Potentially unsafe to access nullable cast expression "<not implemented>" with attribute "foo".
